pkg/controllers/node/termination/terminator/terminator.go

```diff
@@ -72,23 +72,21 @@ func (t *Terminator) Taint(ctx context.Context, node *v1.Node) error {
 		if err := t.kubeClient.Patch(ctx, node, client.MergeFrom(stored)); err != nil {
 			return err
 		}
-		logging.FromContext(ctx).Infof("tainted node")
+		logging.FromContext(ctx).Infof("tainted with: %v", taint)
```

Comment:
```suggestion
		logging.FromContext(ctx).With("taint", taint.Key).Infof("tainted node")
```

Consider making the taint value that you are using here as a structured value

Reply:
Since this is a generic function now, would you agree that it makes sense to include the taint value and effect as well?
```
logging.FromContext(ctx).With("taintKey", taint.Key).With("taintEffect", taint.Effect).With("taintValue", taint.Value).Infof("tainted node")
```